build/localnet/README.md

```diff
@@ -159,6 +159,30 @@ If you need to add a new parameter – feel free to modify the section in place.
 
 Please refer to helm charts documentation for more details.
 
+### Overriding default values for localnet with Tilt
```

Comment:
I do not believe they will vary greatly. It is more for experimentation or adding specific configurations depending on a developers choice of environment. It's a small escape hatch for when you need to deviate a little bit from the defaults.

I only needed to enable CORS for nodes so I could use swagger. It could be used to change any part of the helm chart.

Perhaps you had something else in mind for customizing one's localnet env?